source/loader/layers/sanitizer/stacktrace.hpp

```diff
@@ -18,14 +18,16 @@
 
 namespace ur_sanitizer_layer {
 
-constexpr size_t MAX_BACKTRACE_FRAMES = 64;
+constexpr size_t MAX_BACKTRACE_FRAMES = 48;
```

Comment:
why this change?

Reply:
I originally think 64 is too large, but looks like it will not impact performance. Revert this change.